automl/google/cloud/automl_v1beta1/tables/tables_client.py

```diff
@@ -634,7 +634,8 @@ def delete_dataset(
 
         Returns:
             A :class:`~google.cloud.automl_v1beta1.types._OperationFuture`
-            instance.
+            instance. See the documentation here:
+            https://googleapis.dev/python/google-api-core/latest/operation.html
```

Comment:
If the return type is listed in the public docs, you can list it here (e.g., `google.api_core.retry.Retry`,  `google.api_core.operation.Operation`) and the doc generator will make a link. See [examples](https://googleapis.dev/python/automl/latest/gapic/v1beta1/tables.html).

Reply:
Updated to use above return format & `google.api_core.operation.Operation` class